index.html

```diff
@@ -6,6 +6,6 @@
     <title>Nep huiswerkopdracht</title>
 </head>
 <body>
-
+<p>Dus het is niet echt</p>
```

Comment:
Dit is echt wel echt. I k zie het toch op mijn scherm staan. Misschien moet je er iets anders mee doen? 👍